server/configs/passport-setup.js

```diff
@@ -0,0 +1,45 @@
+import passport from 'passport';
+import Strategy from 'passport-google-oauth20';
+import User from '../models/user';
+import config from './config';
+
+const GoogleStrategy = Strategy;
+// After passsport callback user will be passed here were it will serialize user.id
```

Comment:
I know this is nitpicking, but we must aspire to write the cleanest code with proper spellings.

  passsport -> passport
  were -> where